### ISSUE TITLE
how to annotation text after get the trained ner_cwlm_lstm_crf.model model ?

### ISSUE THREAD

**Issue by cloudtrends** [LiyuanLucasLiu/LM-LSTM-CRF#24]
hi,

After a long time run, the final model file named :  ner_cwlm_lstm_crf.model been generated in the checkpoint folder .
Then the next step is use it as input for annotation some text .
Is there a script called test or something like this can do text or annotation job ?

thanks in advance .

**Comment by LiyuanLucasLiu** [comment]
Thanks for asking, we do provide such kind of scripts. Pls check [ReadMe](https://github.com/LiyuanLucasLiu/LM-LSTM-CRF#prediction)

BTW, GPUs are highly recommended for training and inference. We found that GPUs can bring not only better efficiency but also better effectiveness (in some cases).

**Comment by LiyuanLucasLiu** [comment]
Also, i'd like to mention that we provide some pre-trained models at [ReadMe](https://github.com/LiyuanLucasLiu/LM-LSTM-CRF#pretrained-model)

**Comment by cloudtrends** [comment]
I loss my focus on ReadMe , sorry for brother about this  naive question !

And thanks for kind tips about use GPU . I will try it later.
